src/createList.js

```diff
@@ -1,54 +1,59 @@
-import Checked from "./completed.js";
-import Store from "./store.js";
-import menu from "./menu.svg";
-import deleteList from "./deleteList.js";
+import Checked from './completed.js';
+import Store from './store.js';
+import menu from './menu.svg';
+import deleteList from './deleteList.js';
 
-let todoList = document.querySelector(".todo-list");
+const todoList = document.querySelector('.todo-list');
 
-let store = new Store();
+const store = new Store();
+/* eslint-disable import/no-mutable-exports */
 export let count = 0;
```

Comment:
Kindly, remove all disabled linters, remember linters help us ensure that we write code that is clean and easy to maintain
src/createList.js line 9